src/sentry/middleware/integrations/parsers/base.py

```diff
@@ -19,6 +19,8 @@
 from sentry.types.region import Region, get_region_for_organization
 
 logger = logging.getLogger(__name__)
+if TYPE_CHECKING:
+    from sentry.middleware.integrations.integration_control import ResponseHandler
```

Comment:
if you're fighting import cycles here this might be a good sign to move this type alias to a separate module